app/src/sharedTest/java/org/oppia/android/app/testing/NavigationDrawerTestActivityTest.kt

```diff
@@ -276,93 +255,63 @@ class NavigationDrawerTestActivityTest {
   }
 
   @Test
-  // TODO(#973): Fix NavigationDrawerTestActivityTest
-  @Ignore
-  fun testNavigationDrawerTestActivity_clickNavigationDrawerHamburger_navigationDrawerIsOpenedSuccessfully() { // ktlint-disable max-line-length
+  fun testNavigationDrawerTestActivity_openNavDrawer_navigationDrawerIsOpenedSuccessfully() {
     launch(NavigationDrawerTestActivity::class.java).use {
-      onView(withContentDescription(R.string.drawer_open_content_description))
-        .check(
-          matches(
-            isCompletelyDisplayed()
-          )
-        )
-        .perform(click())
+      it.openNavigationDrawer()
       onView(withId(R.id.home_fragment_placeholder)).check(matches(isCompletelyDisplayed()))
       onView(withId(R.id.home_activity_drawer_layout)).check(matches(isOpen()))
     }
   }
 
   @Test
-  // TODO(#973): Fix NavigationDrawerTestActivityTest
-  @Ignore
   fun testNavigationDrawerTestActivity_clickNavigationDrawerHamburger_changeConfiguration_navigationDrawerIsOpenedSuccessfully() { // ktlint-disable max-line-length
     launch(NavigationDrawerTestActivity::class.java).use {
-      onView(withContentDescription(R.string.drawer_open_content_description))
-        .check(
-          matches(
-            isCompletelyDisplayed()
-          )
-        )
-        .perform(click())
+      it.openNavigationDrawer()
       onView(withId(R.id.home_fragment_placeholder)).check(matches(isCompletelyDisplayed()))
       onView(isRoot()).perform(orientationLandscape())
       onView(withId(R.id.home_activity_drawer_layout)).check(matches(isOpen()))
     }
   }
 
   @Test
-  // TODO(#973): Fix NavigationDrawerTestActivityTest
-  @Ignore
   fun testNavigationDrawerTestActivity_openNavigationDrawerAndRotate_navigationDrawerIsNotClosedAfterRotationIsVerifiedSuccessfully() { // ktlint-disable max-line-length
     launch(NavigationDrawerTestActivity::class.java).use {
-      onView(withContentDescription(R.string.drawer_open_content_description)).perform(click())
+      it.openNavigationDrawer()
       onView(isRoot()).perform(orientationLandscape())
       onView(withId(R.id.home_activity_drawer_layout)).check(matches(isOpen()))
     }
   }
 
   @Test
-  fun testNavigationDrawerTestActivity_openNavigationDrawerAndClose_closingOfNavigationDrawerIsVerifiedSuccessfully() { // ktlint-disable max-line-length
```

Comment:
Per discussion, let's keep this as an Espresso-only test with an explanation of why (e.g. that it's hard to test verifying drawer closing on Robolectric & in general this is mostly testing drawer-specific behavior).

Reply:
okay, I thought we can have only one closing test as it is testing in the clicking on cancel in the switch profile. 
Will add this test back too.

Reply:
Making sure we test different cases of it closing is also important, so I think it's good to add it back. Thanks!